spack.yaml

```diff
@@ -1,8 +1,9 @@
 spack:
   specs:
+    - hpx build_type=Release cxxstd=14 networking=none
+    - boost cxxstd=14 ~atomic~math~iostreams~locale~graph~random~serialization~signals~timer~test~wave~mpi~python
```

Comment:
I'm not sure about this.
Since `boost` is an `HPX` dependency and not our one, shouldn't we rely on `HPX` spack package for this?

Reply:
I can rewrite the above as `hpx ... ^boost ...`, however the line becomes rather long. Since we have `  concretization: together`, the approach is equivalent to the one in the PR.

Reply:
Ok, with the offline chat we had I understasnd why you prefer to explicitly declare the boost dependency like this.